.readthedocs.yaml

```diff
@@ -18,10 +18,12 @@ build:
   os: ubuntu-24.04
   tools:
     python: "3.12"
-
-python:
-  install:
-    - method: pip
-      path: .
-      extra_requirements:
-        - docs
+  jobs:
```

Comment:
Did you test that to build a doc on RTD?

Reply:
Yes, both [here](https://app.readthedocs.com/projects/canonical-starbase/builds/2753533/) and in craft-store.